stack/C/stackUsingArray.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+#include<stdlib.h>
+void main() {
+	void push(int *arr, int *top, int n, int i);
+	int pop(int *arr, int *top);
+	// number of elements
+	int n = 6;
+	int arr[n], top = -1;
+	// push all elements into the stack
+	for(int i = 0; i < n; i++) {
+		push(arr, &top, n, i);
+	}
+	printf("\n");
+	// pop elements one by one.
+	for(int i = 0; i < n; i++) {
+		printf("%d ", pop(arr, &top));
+	 }
+ }
+void push(int *arr, int *top, int n, int i) {
+	if(*top == n)
+		printf("\nStack is full. \n");
+	else
+		arr[++(*top)] = i;
+ }
+int pop(int *arr, int *top) {
+	if(*top == -1) {
+		printf("\n Stack empty!\n");
+		exit(1);
+	 }
```

Comment:
Weird number of spaces at line-start.  Are you using a 2-space indent?  [whitespace/indent] [3]

*Origin: CPPLintBear, Section: `all.cpplint`.*

Reply:
I tried fixing this in f487f893db0005853d256fae370cc15e69c4bd5f commit.